src/NuGet.Core/NuGet.Protocol/Plugins/Messages/GetOperationClaimsRequest.cs

```diff
@@ -9,42 +9,29 @@ namespace NuGet.Protocol.Plugins
 {
     /// <summary>
     /// A query to a plugin about which operations it supports for a specific package source.
+    /// In version 1.0.0, the fields are required. That's not the case for version 2.0.0
     /// </summary>
     public sealed class GetOperationClaimsRequest
     {
         /// <summary>
         /// Gets the package source repository location for the <see cref="ServiceIndex" />.
         /// </summary>
-        [JsonRequired]
         public string PackageSourceRepository { get; }
 
         /// <summary>
         /// Gets the service index (index.json) for the <see cref="PackageSourceRepository" />.
         /// </summary>
-        [JsonRequired]
         public JObject ServiceIndex { get; }
 
         /// <summary>
         /// Initializes a new instance of the <see cref="GetOperationClaimsRequest" /> class.
         /// </summary>
         /// <param name="packageSourceRepository">The package source location.</param>
         /// <param name="serviceIndex">The service index (index.json).</param>
-        /// <exception cref="ArgumentException">Thrown if <paramref name="packageSourceRepository" /> is either
-        /// <c>null</c> or an empty string.</exception>
-        /// <exception cref="ArgumentNullException">Thrown if <paramref name="serviceIndex" /> is <c>null</c>.</exception>
+        /// <remarks>Both packageSourceRepository and service index can be null. If they are, the operation claims request is considered as source agnostic</remarks>
```

Comment:
`request is considered as source agnostic` what does this really mean?

Reply:
GetOperationClaimsRequest takes a source..basically the question is:
- Which operations can you handle for this source?

If both are null, the question effectively changes to:
- Which operations can you handle that are not necessarily source specific? 
In the credential service, we talk in Uris, not sources, so the source concept is non-existent,s so we can't ask that question.